server/index.js

```diff
@@ -7,9 +7,20 @@ const port = 3000;
 const Game = require('../db/index.js');
 
 app.use(cors())
-app.use(express.json())
+app.use(express.json());
+app.use(bodyParser.urlencoded({extended: false}));
```

Comment:
Any particular reason why you are using bodyParser middleware as well as express' native body parsing middleware with express.json?

Reply:
I meant to delete body parser. Its gone now.